### DISCUSSION TITLE
Retroatchivments

### DISCUSSION THREAD

**Discussion by arthur-mts** [eduardofilo/RG350_adam_image#114] [RetroArch topics]
Hello! Does adam image supports RetroArch Retroatchivments on RG350 (with a external wifi dongle of course)

**Comment by eduardofilo** [comment]
Hi.
Well, it is RetroArch the one that doesn't support Retroachievements on its OpenDingux version. Ninoh-FOX compiled a version with Retroachievements enabled for OpenDingux, but he attached to a Discord channel, so I can't point you to a public download link.

**Comment by arthur-mts** [comment]
can you send me a link for this discord server? i will really like to have Retroachievements on my RG350

**Comment by eduardofilo** [comment]
I think it is this: https://discord.gg/wurh4WM

Once inside, the channel where Ninoh-FOX uploaded the file is `Device Releases > od-stock`.
The file was uploaded at 2021-09-21 19:01 (CET or GMT+1).

**Reply by SuperBadger77** [reply]
I finally got a reply from RetroArch. Seems WiFi support now added to the Rg350 image: https://github.com/libretro/RetroArch/issues/14083

**Reply by arthur-mts** [reply]
that's awesome!

**Comment by JazzJohannes** [comment]
Which dongle chipset did you use? I tried it with adam image and a plethora
of chipsets, specifically the one built-in as a driver (digging through
command line)... and none worked, it just wants to connect but keeps
disconnecting... I have the latest RG350P from Amazon (with the fluke
upgrade of RG350M screen built in).

**Reply by arthur-mts** [reply]
I have one esaeger dongle from Ali express, I will try it this weekend

**Reply by SuperBadger77** [reply]
It’s been ages since I tried. I’m sure I did get Wi-Fi working with a dongle and a certain version of firmware but don’t recall the dongle or chipset. If Wi-Fi is now supported by RetroArch as per the link (this support may have been introduced a while back as the PR was some time ago). Don’t we just need the Adam Image to include the updated RetroArch and then we can try Wi-Fi dongles or is it not that simple?

**Reply by arthur-mts** [reply]
honestly i dont know. but, even if you can connect the Linux system to the internet for me the best use case for wireless connection is be able to use retro achievements, and i dont know if the RetroArch version running on the RG350 supports it

**Reply by arthur-mts** [reply]
i know that the "biggest" versions of RetroArch supports it (the ones installed on more powerfull devices) supports it

**Comment by JazzJohannes** [comment]
It has to first actually maintain a connection to WiFi on the Adam Image
OS's side, and only then can RetroArch access it, and last I tried, the OS
kept disconnecting, to my thinking it was completely omitted in Adam's
build-process driver-dependencies wise. But I am interested to get
corrected, as I love my RG350P. BTW the youtube video of instructions to
set-up WiFi, I believe is still this one:
https://www.youtube.com/watch?v=ymhuU_8seXc

**Comment by a-dekker** [comment]
NetPlay is technically working on OD-beta. Tried it with Contra on the NES using the FCEUmm core a while ago (locally on two devices). I did not find it a great experience, some moments the screen started flickering or suffered disconnects/slowdowns after a few minutes. Maybe some other setup/configuration/core will give a more consistent good gameplay experience, I am not really a NetPlay user for that matter.

P.s. I do not see an Achievements option in the settings menu of RetroArch.
